src/esp/gfx/test/CMakeLists.txt

```diff
@@ -1,8 +1,10 @@
 find_package(Magnum REQUIRED Primitives)
 
+if(NOT CMAKE_BUILD_TYPE MATCHES "ASAN")
```

Comment:
Better would be to add a suppression file that matches `libnvidia-eglcore.so` -- we still want to be informed about *our* leaks :) Here's how I do it in magnum: 

- https://github.com/mosra/magnum/blob/master/package/ci/leaksanitizer.conf 
- https://github.com/mosra/magnum/blob/e72a0459c825621beb0298fd76c12b1eeaea8301/package/ci/travis-desktop.sh#L55

Reply:
Cool. Thanks for the pointer. I will try this.